os/linux_base.h

```diff
@@ -123,12 +138,67 @@ typedef void (*dispatch_mach_msg_destructor_t)(void*);
 #endif
 #define OS_NOTHROW
 
+#ifdef OS_ASSUME_NONNULL_BEGIN
```

Comment:
this is not the right definition, doing this will break the overlay, please take the definition from <os/base.h> in an OS X SDK